### PR TITLE
Crow improvements and small cleanup.

### PR DESCRIPTION
**Pull request by Shrimpstress**
Implement wing animations and head animation for crows. Move TreasureChestModel to model package. Remove newlines in multiple biome classes.

### CODE REVIEW

**Review by Shnupbups — APPROVED** [review]

**Comment by Shrimpstress** [comment]
Don't merge just yet, I have another small change I'm going to make.

**Comment by Shrimpstress** [comment]
Alright, this should be good once we get more approvals now.

File: src/main/java/com/fabriccommunity/spookytime/client/model/CrowEntityModel.java [hunk 1]
```diff
@@ -75,15 +74,25 @@ public void setRotationAngle(Cuboid cuboid, float x, float y, float z) {
 		cuboid.yaw = y;
 		cuboid.roll = z;
 	}
-	
+
+	@Override
+	public void animateModel(CrowEntity entity_1, float float_1, float float_2, float float_3) {
+		super.animateModel(entity_1, float_1, float_2, float_3);
+	}
```
Comment (Juuxel) [inline src/main/java/com/fabriccommunity/spookytime/client/model/CrowEntityModel.java]: Unnecessary override?

Comment (Shrimpstress) [reply src/main/java/com/fabriccommunity/spookytime/client/model/CrowEntityModel.java]: Ah, my bad. I thought that was in the class before the PR, guess I added it myself.

**Review by Juuxel — APPROVED** [review]

**Pull request merged**